server/satnogs-web.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
 from app import app
 
+app.config['SECRET_KEY'] = 'the earth is flat'
```

Comment:
Where it is used? Is this value secure? Should we commit it?

Reply:
It's necessary. Otherwise there's exception when flask-login attempts to use session to store successful login. Added text explaining that:

If you're curious, here's the full stack:
```python
venv/lib/python3.7/site-packages/werkzeug/test.py:1016: in post
    return self.open(*args, **kw)
venv/lib/python3.7/site-packages/flask/testing.py:227: in open
    follow_redirects=follow_redirects,
venv/lib/python3.7/site-packages/werkzeug/test.py:970: in open
    response = self.run_wsgi_app(environ.copy(), buffered=buffered)
venv/lib/python3.7/site-packages/werkzeug/test.py:861: in run_wsgi_app
    rv = run_wsgi_app(self.application, environ, buffered=buffered)
venv/lib/python3.7/site-packages/werkzeug/test.py:1096: in run_wsgi_app
    app_rv = app(environ, start_response)
venv/lib/python3.7/site-packages/flask/app.py:2463: in __call__
    return self.wsgi_app(environ, start_response)
venv/lib/python3.7/site-packages/flask/app.py:2449: in wsgi_app
    response = self.handle_exception(e)
venv/lib/python3.7/site-packages/flask/app.py:1866: in handle_exception
    reraise(exc_type, exc_value, tb)
venv/lib/python3.7/site-packages/flask/_compat.py:39: in reraise
    raise value
venv/lib/python3.7/site-packages/flask/app.py:2446: in wsgi_app
    response = self.full_dispatch_request()
venv/lib/python3.7/site-packages/flask/app.py:1951: in full_dispatch_request
    rv = self.handle_user_exception(e)
venv/lib/python3.7/site-packages/flask/app.py:1820: in handle_user_exception
    reraise(exc_type, exc_value, tb)
venv/lib/python3.7/site-packages/flask/_compat.py:39: in reraise
    raise value
venv/lib/python3.7/site-packages/flask/app.py:1949: in full_dispatch_request
    rv = self.dispatch_request()
venv/lib/python3.7/site-packages/flask/app.py:1935: in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
app/controllers/login.py:72: in login
    flash("Invalid username.")
venv/lib/python3.7/site-packages/flask/helpers.py:423: in flash
    session["_flashes"] = flashes
venv/lib/python3.7/site-packages/werkzeug/local.py:350: in __setitem__
    self._get_current_object()[key] = value
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <NullSession {}>, args = ('_flashes', [('message', 'Invalid username.')]), kwargs = {}

    def _fail(self, *args, **kwargs):
        raise RuntimeError(
>           "The session is unavailable because no secret "
            "key was set.  Set the secret_key on the "
            "application to something unique and secret."
        )
E       RuntimeError: The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.

venv/lib/python3.7/site-packages/flask/sessions.py:103: RuntimeError

```